soc/xtensa/intel_adsp/cavs/Kconfig.defconfig.series

```diff
@@ -12,4 +12,12 @@ config INTEL_ADSP_CAVS
 
 source "soc/xtensa/intel_adsp/cavs/Kconfig.defconfig.cavs*"
 
+if GDBSTUB
+
+# GDB expects 420 byte G-packet, so double for hexadecimal digits.
+config GDBSTUB_BUF_SZ
+	default 840 if GDBSTUB
```

Comment:
Is this a kconfig kind of thing?  Seems like a protocol detail outside the control of a zephyr app, no?  If 420 (heh) is required by the upstream toolchain, that's just the number we have to use.  No point in making it configurable, just #define it somewhere.